mixins/localize-dynamic-mixin.js

```diff
@@ -0,0 +1,43 @@
+import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
+import { LocalizeMixin } from './localize-mixin.js';
+
+/*** WARNING! ¡ATENCIÓN! ACHTUNG! Adding to defaultLangs MUST be a BREAKING change with a MAJOR version bump! ***/
```

Comment:
After sleeping on it, I guess this doesn't really help us given the BSI dependency situation. Maybe it's just better we require it to be explicit.

Reply:
Yeah. Like we've really tried hard to avoid major version bumps of core, because coordinating that switch across all the repos that get pulled into BSI is a nightmare -- they all need to upgrade together anyway.

Reply:
Unfortunate, as I was really patting myself on the back for this warning message.

Reply:
Haha it's a fantastic message.

Reply:
Yeah, we would make heroic efforts to avoid a breaking change.  It would be **extremely** disruptive. We should ensure we have a strategy such that we never have to make a major version bump.